hbase-server/src/main/java/org/apache/hadoop/hbase/master/MasterWalManager.java

```diff
@@ -398,4 +414,33 @@ public void archiveMetaLog(final ServerName serverName) {
       LOG.warn("Failed archiving meta log for server " + serverName, ie);
     }
   }
+
+  private static Stoppable createDummyStoppable() {
+    return new Stoppable() {
+      private volatile boolean isStopped = false;
+
+      @Override
+      public void stop(String why) {
+        isStopped = true;
+      }
+
+      @Override
+      public boolean isStopped() {
+        return isStopped;
+      }
+    };
+  }
+
+  public ScheduledChore getOldWALsDirSizeUpdaterChore() {
+    return new ScheduledChore("UpdateOldWALsDirSize", createDummyStoppable(), OLD_WAL_DIR_UPDATE_INTERVAL) {
```

Comment:
Can we make this interval configurable? This way we could also give the option to disable this chore altogether.

Reply:
If we disable it, won't we encounter issues with the incorrect results of the oldWALs directory size JMX metrics? Could that potentially be a problem? That's why I didn't make it configurable.

Reply:
We could return 0 or a negative value in the metric, and document that such values mean the metric calculation is disabled. I think it's worth to make it possible for operators to disable it because this would consume a spot in the ChoreExecutor pool, which could impact execution of other important background work, such as the file cleaners.

Reply:
Got it, will modify the code for that!